### DISCUSSION TITLE
Test Email Button

### DISCUSSION THREAD

**Discussion by avow1337** [sct/overseerr#456] [Feature Requests]
Love what you and the team have done so quickly, impressive stuff! I was just thinking a Test Email button on the Notifications tab would be great as I had to download something to test each time I messed it up.

**Comment by sct** [comment]
This is already released. Update your image.

https://github.com/sct/overseerr/issues/309

**Comment by avow1337** [comment]
Thank you! Looks like I was back on 1.11!
